style.css

```diff
@@ -1,13 +1,926 @@
+/* ----------------- SPEAKERS ------------------- */
+
+.section-speakers {
+  color: #272a31;
+  padding-top: 30px;
+  margin-bottom: 3rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.section-speakers h2 {
+  font-size: 20px;
+  font-weight: 400;
+}
+
+#speaker-grid-container {
+  display: grid;
+  grid-template-columns: auto;
+}
+
+.speaker-full-container {
+  display: flex;
+  margin: 0 2.5% 5px 2.5%;
+  margin-bottom: 20px;
+}
+
+.speaker-picture-container {
+  width: 30%;
+  display: flex;
+  flex-direction: column;
+  position: relative;
+}
+
+.race-flag {
+  width: 65px;
+  position: absolute;
+  float: left;
+  z-index: 1;
+}
+
+.speaker-picture {
+  width: 65%;
+  height: 75%;
+  margin-left: 8%;
+  position: relative;
+  margin-top: 8%;
+  z-index: 10;
+}
+
+.speaker-profile-data {
+  width: 65%;
+  margin-left: 12px;
+}
+
+.speaker-profile-data-name {
+  color: #272a31;
+  font-size: 1rem;
+  font-weight: 700;
+  padding-bottom: 10px;
+}
+
+.speaker-profile-data-role {
+  color: #ec5242;
+  font-size: 0.7rem;
+  font-weight: 700;
+  margin-bottom: 0.6rem;
+  font-style: italic;
+}
+
+.speaker-profile-data-description {
+  color: #272a31;
+  font-size: 0.55rem;
+  font-weight: 700;
+  padding-bottom: 5px;
+}
+
+.down-arrow-icon {
+  width: 1rem;
+  margin-left: 12px;
+}
+
+/* ||||||| ----------------- ABOUT MOBILE SECTION ------------------- |||||||| */
+
+.about-section {
+  background-color: #f8f6f6;
+}
+
+.about-intro-container {
+  margin: 0 2.5% 0 2.5%;
+  display: flex;
+  flex-direction: column;
+  padding-top: 12px;
+  padding-bottom: 55px;
+  color: #ec5242;
+  justify-content: center;
+}
+
+.about-intro-container h2 {
+  font-size: 1.4rem;
+  font-weight: 500;
+  margin-bottom: 18px;
+  align-self: center;
+  margin-top: 78px;
+  text-align: center;
+}
+
+.about-intro-title-conatiner h1 {
+  font-size: 2.3rem;
+  font-weight: 900;
+  text-align: center;
+}
+
+.about-description-container {
+  background-color: #fff;
+}
+
+.about-description {
+  margin: 0 2.5% 0 2.5%;
+  padding: 1.5rem 0;
+  line-height: 1.5rem;
+  text-align: justify;
+}
+
+.about-further-questions-container {
+  background-color: #f8f6f6;
+  padding: 25px 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.about-further-questions-paragraph {
+  color: #272a31;
+  font-size: 0.75rem;
+  font-weight: 700;
+  line-height: 1.4rem;
+}
+
+.about-contact-mail {
+  color: #272a31;
+  font-size: 0.75rem;
+  font-weight: 900;
+}
+
+/* *********** Logos Section ************* */
+
+.logo-creation {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.logo-creation-title {
+  align-self: center;
+  padding-top: 1.5rem;
+  padding-bottom: 1rem;
+  margin: 0 15% 0 15%;
+  font-weight: 700;
+  font-size: 20px;
+  color: #272a31;
+}
+
+.logo-creation p {
+  font-weight: 700;
+  font-size: 1rem;
+  color: #d3d3d3;
+  line-height: 25px;
+  margin: 0 2.5% 1.5rem 2.5%;
+  text-align: justify;
+}
```

Comment:
- [ ] I think the color here is a bit light than how it is present in the design guidelines. Please make sure to fix this by following design guidelines 🙌
- [ ] Can you please reduce the `margin` between the respective elements a bit? 🤓⛏
- [ ] Please make sure to align the text to center for mobile version 📱 Also, can you please fix the border color of the below mentioned elements (mobile version)


### **_Desktop version_**
Your design | Design guidelines
---| ---|
![image](https://github.com/HFG43/Capstone1/assets/106119105/dbf1f520-e87f-4c05-9b0d-bd746dcab14d) | ![image](https://github.com/HFG43/Capstone1/assets/106119105/ba57b3d7-441e-4dbb-8c0c-ab1fc350ff5f)

### **_Mobile version_**
Your design | Design guidelines
---| ---|
![image](https://github.com/HFG43/Capstone1/assets/106119105/02802b70-0a10-4206-b61a-56474427f02c) | ![image](https://github.com/HFG43/Capstone1/assets/106119105/7ce7e67f-b4a5-48fc-a357-55f92dfbc42d)